spec/controllers/users/dossiers_controller_spec.rb

```diff
@@ -464,6 +464,17 @@
         end
       end
 
+      context "with dossier created on procedure routee but routage disabled later" do
+        let!(:dossier) { create(:dossier, groupe_instructeur: nil, user: user) }
```

Comment:
Un `let` devrait être suffisant, non ?

```suggestion
        let(:dossier) { create(:dossier, groupe_instructeur: nil, user: user) }
```